.ci/releasenotes-gen-xdoc-push.sh

```diff
@@ -4,6 +4,15 @@ set -e
 source ./.ci/util.sh
 checkForVariable "READ_ONLY_TOKEN"
 
+if [[ -z $1 ]]; then
+  echo "version is not set"
+  echo "Usage: update-releasenotesxml.sh <version>"
```

Comment:
Fix name of script

Reply:
```
~/java/github/romani/checkstyle [stoyanK7/issue/12488 L|✚ 1] 
$ git diff
diff --git a/.ci/releasenotes-gen-xdoc-push.sh b/.ci/releasenotes-gen-xdoc-push.sh
index 8056cd9..32856e3 100755
--- a/.ci/releasenotes-gen-xdoc-push.sh
+++ b/.ci/releasenotes-gen-xdoc-push.sh
@@ -2,11 +2,11 @@
 set -e
 
 source ./.ci/util.sh
-checkForVariable "READ_ONLY_TOKEN"
+#checkForVariable "READ_ONLY_TOKEN"
 
 if [[ -z $1 ]]; then
   echo "version is not set"
-  echo "Usage: update-releasenotesxml.sh <version>"
+  echo "Usage: $(basename $0) <version>"
   exit 1
 fi
 
✔ ~/java/github/romani/checkstyle [stoyanK7/issue/12488 L|✚ 1] 
$ .ci/releasenotes-gen-xdoc-push.sh
version is not set
Usage: releasenotes-gen-xdoc-push.sh <version>

```

Reply:
fixed

Reply:
Uh, realised that mistake just after commiting. Thanks for fixing it.